ivy/functional/frontends/numpy/sorting_searching_counting/searching.py

```diff
@@ -84,6 +84,5 @@ def nanargmax(a, /, *, axis=None, out=None, keepdims=False):
 
 @to_ivy_arrays_and_back
 @from_zero_dim_arrays_to_scalar
-def nanargmin(a, /, *, axis=None, out=None, keepdims=False):
```

Comment:
`nanargmin` is still missing from your code. If you look at the check PR `numpy-frontends-test -> Details -> Run Frontend Tests`, you'll see that it's not running due to a KeyError in the corresponding test function (can't find the function to test)
```
==================================== ERRORS ====================================
_ ERROR collecting ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py _
ivy_tests/test_ivy/test_frontends/test_numpy/test_sorting_searching_counting/test_searching.py:373: in <module>
    def test_numpy_nanargmin(
ivy_tests/test_ivy/helpers/testing_helpers.py:251: in test_wrapper
    callable_fn, fn_name, fn_mod = _import_fn(fn_tree)
ivy_tests/test_ivy/helpers/testing_helpers.py:131: in _import_fn
    callable_fn = mod.__dict__[fn_name]
E   KeyError: 'nanargmin'
```